Hummingbird.docc/Tutorials/Todos/Resources/code/Postgres/todos-postgres-10.swift

```diff
@@ -33,3 +33,18 @@ public func buildApplication(_ arguments: some AppArguments) async throws -> som
     return app
 }
 
+/// Build router
```

Comment:
The tutorial text here is describing the fact we need a background service for the connection manager. It isn't necessary to display `buildRouter()` here.

Instead it'd be better to make it clear we are only displaying `buildApplication()`. If you go into the Todos-4-Postgres.tutorial and find where we are displaying this code and set the title to be "buildApplication() - Sources/App/Application+build.swift" instead of "Sources/App/Application+build.swift".